src/main/pages/che-7/overview/con_glossary.adoc

```diff
@@ -0,0 +1,43 @@
+---
+title: Che glossary
+keywords:
+tags: []
+sidebar: che_7_docs
+permalink: che-7/che-glossary/
+folder: che-7/overview
+summary:
+---
+
+:page-liquid:
+:parent-context-of-che-glossary: {context}
+
+[id="{prod-id-short}-glossary_{context}"]
+= {prod-short} Glossary
+
+:context: {prod-id-short}-glossary
+
+== Naming
+{prod}:: {prod} is a developer platform for the cloud that provides an in-browser IDE. {prod-short} tools can be extended by adding development services, such as language servers, debug adapters, or editors (IDEs) packaged as containers. 
+
+Che-Theia:: Che-Theia is the default editor component of a {prod-short} workspace. It is a {prod-short}-specific customization of the Eclipse Theia editor. Che-Theia can be extended using Che-Theia plugins, which are packaged as containers for ease of deployment and use. Che-Theia plugins are mostly compatible with Visual Studio extensions.
+
+{prod-short} Workspace:: {prod-short} Workspace is a container-based development environment managed by {prod}. Every {prod-short} workspace is defined by a devfile. A {prod-short} workspace is composed by an editor, some runtime and build containers and other development tools running inside containers.
+
+== Terminology
+Organization:: Organizations in {prod} allow administrators to group {prod} users and allocate resources. The system administrator controls and allocates resources and permissions within the administrator dashboard.
+
+Editor:: Editor is a web application that is used as an editor in a workspace.
+
+Plugin:: {prod-short} plugins are services that extend {prod-short} workspace capabilities. {prod-short} plugins are packaged as containers. Plugins are extensions of an editor or a service running in the container. For example, the Che-Theia editor is compatible with Visual Studio Code extensions. 
+//TODO See for a diagram of Che extensibility architecture. 
+Both {prod-short} plugins and editors are distributed through the {prod-short} plugin registry. 
+ 
+Workspace:: A container based development environment managed by Eclipse Che. Every  {prod-short} workspace is defined by a Devfile. A  {prod-short} workspace can be composed by an editor, some plugins and runtime containers. Workspace runtime containers can be defined as simple container images or as Kubernetes and OpenShift resources. A {prod-short} Workspace can be associated with source code projects hosted on a remote CVS server. A {prod-short} Workspace can contain the definition of one or more commands such as `run`, `build`, or `debug`.
+
+Devfile:: A devfile is a workspace configuration template. Devfiles are used to create workspaces in the Dashboard. A devfile includes meta information such as scope, tags, components, description, name, and identification. VS Code API is one of the plugin APIs that can be used in Theia.
```

Comment:
Should "meta information" be "metadata"?

Also, no need for "A devfile is", similar to above. It's implied by the definition list. :)